src/common/snippets/include/snippets/lowered/pass/split_loops.hpp

```diff
@@ -35,8 +35,24 @@ class SplitLoops : public RangedPass {
     SplitLoops();
     bool run(LinearIR& linear_ir, lowered::LinearIR::constExprIt begin, lowered::LinearIR::constExprIt end) override;
 
+    static void split(LinearIR& linear_ir, size_t loop_to_split_id, size_t outer_increment);
```

Comment:
It's not a particularly good practice to expose implementation details just to alleviate testing.
You sure we can't test the whole pass? It would also be more correct methodologically: we use the whole pass in production,  not only `split()`. It means that the whole pass should be tested.
What do you think?

Reply:
Decided to apply in this PR.